tsconfig.configs.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "noEmit": true
+  },
+  "files": [
+    "./karma.conf.js",
+    "./gulpfile.js"
+  ]
+}
```

Comment:
This particular file is a fake TypeScript project that includes the two configuration files that the project has that happen to be JavaScript (as opposed to JSON/YAML, for example). Because the typescript-eslint parser requires them to be in a TypeScript project. And because I feel they should be subject to linting, as well.

I do this in all my projects. Seems to be a :+1: pattern.